src/stores/CurrentGameStore.ts

```diff
@@ -89,4 +89,21 @@ export class CurrentGameStore {
     }
     return yourHands.slice(-1)[0]
   }
+
+  //Calculate if only one player, exactlly the last one has to announce the tricks
+  @computed get isLastPlayerAnnouncing() {
+    let nullValues = 0
+    this.activeMatch?.hands.forEach(
+      (el) => el.announcedScore === null ?? (nullValues += nullValues + 1)
+    )
+    if (nullValues === 1) return true
+    else return false
```

Comment:
```suggestion
    return this.activeMatch?.hands.filter((el) => el.announcedScore === null).length === 1
```
(Nicht getestet)

Plus ohne anfänger 
if (condition) return true else return false

Reply:
Finde meine Lösung genau so gut...:-)